server/sonar-server/src/test/java/org/sonar/server/user/ws/DeactivateActionTest.java

```diff
@@ -203,7 +203,7 @@ public void fail_to_deactivate_last_administrator_of_organization() throws Excep
     logInAsSystemAdministrator();
 
     expectedException.expect(BadRequestException.class);
-    expectedException.expectMessage("User is last administrator of organizations [org1, org2], and cannot be deactivated");
+    expectedException.expectMessage("User 'test' is last administrator of organizations [org1, org2], and cannot be deactivated");
```

Comment:
Fabrice confirm that the point you've raised it related to the MMF about closing its own account